regress/pesterTests/Authorized_keys_fileperm.Tests.ps1

```diff
@@ -61,8 +61,7 @@ Describe "Tests for authorized_keys file permission" -Tags "CI" {
             Copy-Item $Source $ssouserSSHProfilePath -Force -ErrorAction Stop
 
             Repair-AuthorizedKeyPermission -Filepath $authorizedkeyPath -confirm:$false
-
-            Get-Process -Name sshd  -ErrorAction SilentlyContinue | Where-Object {$_.SessionID -ne 0} | Stop-process -force -ErrorAction SilentlyContinue
+            
```

Comment:
we need to call Stop-SSHDTestDaemon to stop sshd?

Reply:
yes, see the test cases. after we move the debug more process to run under system, this line is useless.